client/src/Views/NotFoundPage/NotFoundPage.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <main class="content">
+    <h1 class="not-found-heading">Not Found</h1>
+    <section class="not-found-info">
+      <p class="not-found-info__subtext">Sorry, we couldn't find the page you were looking for.</p>
+      <p class="not-found-info__subtext">¯\_(ツ)_/¯</p>
+    </section>
+  </main>  
+</template>
+
+<script>
+export default {
+  name: 'Not Found',
+}
+</script>
+
+<style lang="scss" scoped>
+@use "@/assets/styles/variables.scss" as v;
+
+.content {
+  height: 60vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.not-found-heading {
+  color: v.$main-white;
+  font-size: 5rem;
+  font-weight: 800;
+  font-family: "Fjalla One", "Open Sans", sans-serif;
```

Comment:
Also, this font is in the variables file, though at the time I only thought that it was for the navbar...so perhaps renaming it to `$main-font` may be a good idea...